edge-agent/src/Microsoft.Azure.Devices.Edge.Agent.Service/modules/KubernetesModule.cs

```diff
@@ -189,7 +189,8 @@ protected override void Load(ContainerBuilder builder)
             // KubernetesRuntimeInfoProvider
             builder.Register(c => new KubernetesRuntimeInfoProvider(this.deviceNamespace, c.Resolve<IKubernetes>(), c.Resolve<IModuleManager>()))
                 .As<IRuntimeInfoProvider>()
-                .As<IRuntimeInfoSource>();
+                .As<IRuntimeInfoSource>()
+                .SingleInstance();
```

Comment:
Did you mean to add SingleInstance?

Reply:
Checking b/c it seems unrelated to PR :)

Reply:
Yes, this was a bug discovered testing this.  Can't prove I've fixed the problem without this change.